mapie/tests/test_quantile_regression.py

```diff
@@ -519,6 +519,120 @@ def test_linear_regression_results(strategy: str) -> None:
     np.testing.assert_allclose(coverage, COVERAGES[strategy], rtol=1e-2)
 
 
+def test_quantile_prefit_non_list() -> None:
+    """
+    Test that there is a list of estimators provided when cv='prefit'
+    is called for MapieQuantileRegressor.
+    """
+    with pytest.raises(
+        ValueError,
+        match=r".*Estimator for prefit must be an iterable object.*",
+    ):
+        not_an_iterable = 10
```

Comment:
You can parametrize the argument "not_an_iterable" to test different possibilities.

Reply:
Do you have ideas of what other types of non-iterables I could test?

Reply:
A float, a string ?

Reply:
A string is iterable :(. I have added float. Also do we need to add this test both in `test_quantile_regression.py` and `test_utils.py`?

Reply:
Only one in `test_quantile_regression` is enough.